catalyst_voices/lib/pages/account/account_role_ext.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:flutter/material.dart';
+
+extension AccountRoleExt on AccountRole {
```

Comment:
This will be needed in more places, I would put it next to space_ext, in catalyst_voices/lib/common/ext/acount_role_ext.dart